kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -305,10 +305,10 @@ public class BigQuerySinkConfig extends AbstractConfig {
           + "front of field name. Note: field a.b and a_b will have same value after sanitizing, "
           + "and might cause key duplication error.";
   private static final ConfigDef.Type KAFKA_KEY_FIELD_NAME_TYPE = ConfigDef.Type.STRING;
-  private static final ConfigDef.Validator KAFKA_KEY_FIELD_NAME_VALIDATOR = new ConfigDef.NonEmptyString();
   private static final ConfigDef.Importance KAFKA_KEY_FIELD_NAME_IMPORTANCE = ConfigDef.Importance.LOW;
   private static final String KAFKA_KEY_FIELD_NAME_DOC = "The name of the field of Kafka key. "
-      + "Default to be null, which means Kafka Key Field will not be included.";
+      + "Default to be null, which means Kafka Key Field will not be included. "
+      + "To include all fields from the key in the top-level record, specify a blank string for this property.";
```

Comment:
This is kind of a hack. Do we maybe want to consider some other options for enabling this kind of functionality? Specially-defined behavior reliant on an empty string being explicitly specified in a config file is a bit of an anti-pattern.